enhancements/proxy/global-cluster-egress-proxy.md

```diff
@@ -144,108 +143,92 @@ The information can also be modified at runtime.  If it is modified at runtime,
-means they can update their proxy configuration to something that does not work, in the future, if they do
-not add validation endpoints on day 2.  We should consider enhancing the install config to allow specification
-of validation endpoints so this step is not forgotten by administrators.  In addition install is a lengthy
-operation.  Validating the proxy configuration up front would allow us to "fail fast".
-* Currently the no proxy value in the proxy config is append-only.  There is no way for an administrator to
-remove one of the no proxy domains that we add automatically.  This means we must be extremely cautious to
-not add noproxy domains that might need to be proxied.
-* Long term it should be possible for the additional CA bundle to be the *only* source of CAs for components.  
-Today the additional CA bundle is a combination of user provided CAs plus the system CAs from the network 
-operator image.  In the future adding the system trusts to the bundle should be a configurable optiona so customers 
-who want to explicitly control the trusted CAs can do so.
+  the proxy configuration during install.  If it is incorrect, the install will fail.  However it also
+  puts administrators at risk of never providing validation endpoints to their proxy configuration, which
+  means they can update their proxy configuration to something that does not work, in the future, if they do
+  not add validation endpoints on day 2.  We should consider enhancing the install config to allow specification
+  of validation endpoints so this step is not forgotten by administrators.  In addition install is a lengthy
+  operation.  Validating the proxy configuration up front would allow us to "fail fast".
+* Currently the no-proxy value in the proxy config is append-only.  There is no way for an administrator to
+  remove one of the no-proxy domains that we add automatically.  This means we must be extremely cautious to
+  not add no-proxy domains that might need to be proxied.
+* Long term it should be possible for the additional CA bundle to be the *only* source of CAs for components.
+  Today the additional CA bundle is a combination of user provided CAs plus the system CAs from the network
+  operator image.  In the future adding the system trusts to the bundle should be a configurable optiona so customers
+  who want to explicitly control the trusted CAs can do so.
 * In addition, due to 4.1->4.2 upgrade limitations, components must fallback to using their own CAs from their image in the event
-that the configmap does not have a bundle injected into it because the network operator is not upgraded yet.  This
-latter limitation can be removed in 4.3, meaning components can make the configmap key a required mount and expect
-that it will have sufficient content to supplant any system CAs in their own image.
-
-
-See also: [proxy workflow](https://docs.google.com/document/d/1y0t0yEOSnKc4abxsjxEQjrFa1AP8iHcGyxlBpqGLO08/edit#
-).
```

Comment:
i know it's bad form to link to a non-public doc, but there is a lot of useful information in this doc, i'd prefer to retain the link (or convert it to a public file).

Reply:
+1 to making it public :)

Reply:
But I don't have time to work it in now, so I've pushed d8ff771 -> 031097d to just leave this line alone.